engine/runner/src/main/java/org/enso/runner/Repl.java

```diff
@@ -0,0 +1,57 @@
+package org.enso.runner;
+
+import org.enso.polyglot.debugger.ReplExecutor;
+import org.enso.polyglot.debugger.SessionManager;
+import scala.runtime.Nothing$;
+
+final class Repl implements SessionManager {
+  private final ReplIO replIO;
+
+  Repl(ReplIO replIO) {
+    this.replIO = replIO;
+  }
+
+  @Override
+  public Nothing$ startSession(ReplExecutor executor) {
```

Comment:
`Nothing$` ? `void` is perfectly fine

Reply:
That results in compilation error: "return type void is not compatible with Nothing$" - method does not override or implement a method from a supertype